src/custom_nodes/horizontal_ocr/Dockerfile.ubuntu

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM ubuntu:20.04
+RUN apt update && apt install -y build-essential curl
+RUN curl -s https://download.01.org/opencv/master/openvinotoolkit/thirdparty/linux/opencv/opencv_4.5.2-076_ubuntu20.txz | tar --use-compress-program=xz -xvf -
+ARG NODE_NAME=horizontal_ocr
+
+COPY . /custom_nodes/${NODE_NAME}/
+COPY custom_node_interface.h /
+WORKDIR /custom_nodes/${NODE_NAME}/
+RUN mkdir -p /custom_nodes/lib
+RUN g++ -c -std=c++17 ${NODE_NAME}.cpp -fpic  -I/opencv/include/ -Wall -Wno-unknown-pragmas -Werror -Wno-error=sign-compare -fno-strict-overflow -fno-delete-null-pointer-checks -fwrapv -fstack-protector
```

Comment:
Adding those gcc options https://github.com/openvinotoolkit/model_server/pull/1012/files improved the performance in the custom loader. Could you test if if also improve the processing in that custom node lib?